client/src/components/Chatbot/index.js

```diff
@@ -209,11 +80,11 @@ export class Chatbot extends Component {
   }
 
   render() {
-    const { messages, show } = this.state;
+    const { messages, show } = this.props;
 
     return (
       <div className={`container chatbot-container ${this.getContainerClass()}`}>
-        <div className="collapse-header blue" onClick={this.changeShowMessages}>
+        <div className="collapse-header blue" onClick={this.props.onToggleShow}>
```

Comment:
You can add `onToggleShow` to object desctructuring above.

Reply:
True! I did not notice that! Thanks! 👍